CMakeLists.txt

```diff
@@ -274,30 +278,61 @@ IF(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     SET_TARGET_PROPERTIES(${PROJECT_TARGET_BIN} PROPERTIES MACOSX_BUNDLE_INFO_PLIST "${CMAKE_SOURCE_DIR}/resources/Info.plist")
 ENDIF()
 
+# Fixes for Visual Studio 2017 immediate build\execution\debug directly opening CMake folder without project
+# Initialize IS_VS_2017CMAKE to 0, it will be override by Visual Studio cmake with -D command line parameter
+SET(IS_VS_2017CMAKE 0 CACHE BOOL "Executed from VS2017 IDE")
+IF(IS_VS_2017CMAKE EQUAL 1)
+    SET(CONFIGURATION_NAME "${TARGET_ARCH}-${CMAKE_BUILD_TYPE}")
+    SET(EXECUTABLE_OUTPUT_PATH "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}/bin")
+    SET(LIBRARY_OUTPUT_PATH "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}/bin/lib")
+    SET(INSTALL_BIN_DIR "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}/bin")
+    SET(INSTALL_DOC_DIR "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}")
+    SET(INSTALL_RES_DIR "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}/res")
+    SET(INSTALL_MO_DIR "${CMAKE_SOURCE_DIR}/bin/${CONFIGURATION_NAME}/po/en")
```

Comment:
They are neeeded to obtain the same behaviour of old copy_files.bat (https://github.com/moneymanagerex/moneymanagerex/blob/v1.2.x/build/msw-vc-2013e/copy_files.bat) that will be automatically executed by VS after building but before running EXE in debug mode